package.json

```diff
@@ -53,15 +53,15 @@
         "@types/long": "^4.0.2",
         "@types/minimist": "^1.2.2",
         "@types/mocha": "^9.1.1",
-        "@types/node": "^18.0.0",
+        "@types/node": "^22.7.5",
```

Comment:
You can bump this to a higher minor or patch version of 18.x, but we need the major version to stay 18 as that is our minimum supported node.js version